apps/services/auth/public-api/src/app/modules/delegations/actorDelegations.controller.ts

```diff
@@ -8,6 +8,7 @@ import {
 import { ApiTags } from '@nestjs/swagger'
 
 import {
+  DEFAULT_DOMAIN,
```

Comment:
Can revert this and the redundant `await` below.

Reply:
Fixed.